frontend/src/lib/candidate/templates/login/LoginPage.svelte

```diff
@@ -55,19 +57,47 @@ Candidate login page. This component also takes care of the login process.
       autocomplete="email"
       required />
     <label for="password" class="hidden">{$_('candidate.password')}</label>
-    <input
-      type="password"
-      name="password"
-      id="password"
-      class="input mb-md w-full max-w-md"
-      placeholder={$_('candidate.password_placeholder')}
-      bind:value={password}
-      autocomplete="current-password"
-      required />
+
+    {#if passwordRevealed === true}
+      <div class="relative mb-md w-full max-w-md">
+        <input
+          type="text"
+          name="password"
+          id="password"
+          class="input mb-md w-full max-w-md"
+          placeholder={$_('candidate.password_placeholder')}
+          bind:value={password}
+          autocomplete="current-password"
+          required />
+        <Button
+          text=""
+          title="hide the button"
+          class="absolute inset-y-0 right-0 mb-md max-w-md"
+          icon="hide"
+          on:click={() => (passwordRevealed = !passwordRevealed)} />
```

Comment:
Add `variant="icon"` to create icon button with this component. Then `title` can be removed because this component uses value of `text` as `title` when using icon variant. Text would probably be better as something like "Hide the password".